content/cloud-docs/api-docs/state-version-outputs.mdx

```diff
@@ -153,3 +153,65 @@ curl \
   }
 }
 ```
+
+## Show Current State Version Outputs for a Workspace
```

Comment:
What are version outputs? 🤔

Reply:
Another, slightly related question, why would someone use this API?

Reply:
Excellent question! This is how I understand it (anyone feel free to correct me on this): when you initiate a successful run in TFC/E, the resulting Terraform state file is versioned... what we call a state version. Like a regular Terraform statefile, it can have outputs, e.g:
```hcl
# somewhere in my configuration
output "my-ec2-private-ip-addrress" {
  value = aws_instance.server.private_ip
  sensitive = false
}
```
When we talk about state version outputs, we refer to these values. Here are the docs if my explanation isn't clear: https://www.terraform.io/language/values/outputs

Hope this helps! 😃